src/cascadia/TerminalSettingsModel/IInheritable.h

```diff
@@ -72,11 +72,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
     // This is like std::optional, but we can use it in inheritance to determine whether the user explicitly cleared it
     template<typename T>
-    struct NullableSetting
-    {
-        std::optional<T> setting{ std::nullopt };
-        bool set{ false };
-    };
+    using NullableSetting = std::optional<std::optional<T>>;

```

Comment:
@miniksa will have a laugh here

Reply:
I thought we agreed this was gonna be `til::super_optional<T>` or `til::optional_no_really<T>`. :P

Reply:
If we have a need for it outside of TSM, absolutely ;)